File: src/data/ListDay.java
package data;

import java.io.Serializable;

public class ListDay implements Serializable {
  private Appointment[] appointments;
  private Therapist[] therapists;

  public ListDay(Appointment[] appointments, Therapist[] therapists) {
    this.appointments = appointments;
    this.therapists = therapists;
  }

  public Appointment[] getAppointments() {
    return appointments;
  }

  public void setAppointments(Appointment[] appointments) {
    this.appointments = appointments;
  }

  public Therapist[] getTherapists() {
    return therapists;
  }

  public void setTherapists(Therapist[] therapists) {
    this.therapists = therapists;
  }

}

File: frontend/src/class/ListDay.ts
import Appointment from './Appointment';
import Therapist from './Therapist';

export default class ListDay {
  appointments : Appointment[];

  therapists : Therapist[];

  constructor(
    appointments : Appointment[],
    therapists: Therapist[],
  ) {
    this.appointments = appointments;
    this.therapists = therapists;
  }
}
